lib/rubocop/cop/lint/redundant_cop_disable_directive.rb

```diff
@@ -26,6 +26,7 @@ module Lint
       #   # good
       #   x += 1
       class RedundantCopDisableDirective < Base # rubocop:todo Metrics/ClassLength
+        # rubocop:enable Lint/RedundantCopDisableDirective
```

Comment:
Ideally, we'd be able to have this comment appear in same block as above, but we'd need some way to strip it out, and I'm not sure if that exists.